packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-form.tsx

```diff
@@ -20,10 +20,10 @@ import FormSubHeader from 'Components/form-sub-header';
 import FileUploaderContainer from '../../../Components/file-uploader-container';
 import { getFileUploaderDescriptions } from '../../../Constants/file-uploader';
 import { isServerError } from 'Helpers/utils';
-import { income_status_codes, getPoincDocumentsList } from 'Sections/Verification/ProofOfIncome/proof-of-income-utils';
+import { getPoincDocumentsList } from 'Sections/Verification/ProofOfIncome/proof-of-income-config';
 
 type TProofOfIncomeForm = {
-    onSubmit: (status: typeof income_status_codes[keyof typeof income_status_codes]) => void;
+    onSubmit: (status: typeof AUTH_STATUS_CODES[keyof typeof AUTH_STATUS_CODES]) => void;
```

Comment:
Can u add this type in packages/account/src/Types/common.type.ts. It can be reused. I believe in some PR its part of the common types file